arparser/src/parser.sk

```diff
@@ -0,0 +1,372 @@
+module ARParser;
+
```

Comment:
Are there any references to the format specifications that could be added in comments here?

Reply:
Sadly there is no such thing as a standardized spec for this format. The FreeBSD man page is the best resource I found: https://man.freebsd.org/cgi/man.cgi?query=ar&sektion=5.

Reply:
I added a description of the format.